SPECS/python3/python3.spec

```diff
@@ -207,27 +195,27 @@ export OPT="%{extension_cflags} %{openssl_flags}"
 # NOTE: This is a NO-OP for the toolchain build.
 python3 Lib/ensurepip
 
-# Installing pip/setuptools via ensurepip fails in our toolchain.
+# Installing pip via ensurepip fails in our toolchain.
 # The versions of these tools from the raw toolchain are detected,
 # and install fails. We will install these two bundled wheels manually.
 # https://github.com/pypa/pip/issues/3063
 # https://bugs.python.org/issue31916
-pushd Lib/ensurepip/_bundled
-pip3 install --no-cache-dir --no-index --ignore-installed \
-    --root %{buildroot} \
-    setuptools-%{setuptools_version}-py3-none-any.whl \
-    pip-%{pip_version}-py3-none-any.whl
+pip3 install --no-cache-dir --no-index --ignore-installed --root %{buildroot} \
+    ./Lib/ensurepip/_bundled/pip-%{pip_version}-py3-none-any.whl
+
+# Install setuptools
```

Comment:
if upstream python has removed the vendored setuptools (which is great, because using vendored code is pretty awful, as @eric-desrochers knows) I really don't think we should be *re-vendoring* it here. It should be its own package.

Reply:
Sounds fine, I will create a separate package for setuptools